hack/checkmod.sh

```diff
@@ -26,7 +26,7 @@ RELEASES+=("main")
 GOMOD+=("go.mod")
 
 REPO+=("https://raw.githubusercontent.com/k8snetworkplumbingwg/ovs-cni")
-RELEASES+=("main release-0.31 release-0.29")
+RELEASES+=("main release-0.36 release-0.34 release-0.32 release-0.31 release-0.29")
 GOMOD+=("go.mod")
 
 REPO+=("https://raw.githubusercontent.com/k8snetworkplumbingwg/kubemacpool")
```

Comment:
@RamLavi can you suggest please what are the branches we need to monitor for KMP so i will update it ?